src/test/java/org/opentripplanner/routing/algorithm/mapping/CarSnapshotTest.java

```diff
@@ -67,7 +68,7 @@ public void directCarPark() {
     // TODO: 2022-08-30 VIA: Previously we were using RequestModesBuilder
     // maybe we should implement similar pattern for new models?
     request.journey().direct().setMode(StreetMode.CAR_TO_PARK);
-    request.journey().transit().setModes(List.of());
+    request.journey().transit().setFilters(List.of(ExcludeAllTransitFilter.of()));
```

Comment:
If this is also done in non-test code, then we can add a convenience method for it.

Reply:
It's only used for the tests. I moved ExcludeAllTransitFilter to test package.